backend/src/data-source.ts

```diff
@@ -0,0 +1,18 @@
+import { DataSource } from "typeorm";
+import { config } from "dotenv";
+import { User } from "./entity/testEnt";
+config({ path: 'src/env/.env' });
+
+export const AppDataSource = new DataSource({
+    type: "postgres",
+    host: "localhost",
+    port: Number(process.env.port),
+    username: process.env.User_Name,
```

Comment:
there is a convention to give names to constants in upper case: PORT, HOST, USER_NAME and etc
update respectively